packages/node/src/nodeVersion.ts

```diff
@@ -0,0 +1,3 @@
+import { parseSemver } from '@sentry/utils';
+
+export const NODE_VERSION: ReturnType<typeof parseSemver> = parseSemver(process.versions.node);
```

Comment:
l: Should we not put our own type here (not sure what `parseSemver` actually returns, a string??)? Seems a bit redundant and if the return type changes we won't actually find out and it will just be different from before?

Reply:
So I had to include this because otherwise I was getting a build error:

```
created build/esm in 355ms
src/nodeVersion.ts:3:14 - error TS4023: Exported variable 'NODE_VERSION' has or is using name 'SemVer' from external module "/Users/abhijeetprasad/workspace/sentry-javascript/packages/utils/build/types/misc" but cannot be named.

3 export const NODE_VERSION = parseSemver(process.versions.node);
```

It seems because `SemVer` is not exported, the type breaks?